src/scss/components/selector-strip/_index.scss

```diff
@@ -1,127 +1,2 @@
-@import "vars";
```

Comment:
All styles moved to a partial to avoid bringing in more styles than necessary if an external file imports selector strip styles, in the future.